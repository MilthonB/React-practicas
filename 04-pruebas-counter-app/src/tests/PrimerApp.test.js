import { render } from "@testing-library/react";

// import "@testing-library/jest-dom/extend-expect";

import { shallow } from 'enzyme';


import PrimeraApp from "../PrimerApp";



describe('Pruebas a PrimerApp.js', () => {


    // test('debe de retornar un Hola soy goku', () => {

    //     const saludo = 'Hola soy goku';

    //     const { getByText } = render(<PrimeraApp saludo = {saludo} />);
        
    //      expect(getByText(saludo)).toBeInTheDocument();


    // });


    test('debe de mostrar <PrimerApp /> correctamente', () => {


        const saludo = 'Hola soy goku!!';
        const wrapper = shallow( <PrimeraApp saludo={saludo} /> )

         expect(wrapper).toMatchSnapshot();

        // const { getByText } = render(<PrimeraApp saludo = {saludo} />);
        
        //  expect(getByText(saludo)).toBeInTheDocument();


    });


});