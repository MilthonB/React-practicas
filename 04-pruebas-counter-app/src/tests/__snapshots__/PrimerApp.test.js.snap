// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas a PrimerApp.js debe de mostrar <PrimerApp /> correctamente 1`] = `
<Fragment>
  <pre>
     
    Hola soy goku
     
  </pre>
  <p>
     
    Soy un subtitulo
     
  </p>
</Fragment>
`;
